packages/docz-tools/package.json

```diff
@@ -13,14 +13,16 @@
     "@emotion/react": "^11.1.4",
     "@jobber/components": "^3.0.2",
     "@jobber/design": "^0.28.2",
+    "@types/uuid": "^8.3.3",
     "classnames": "^2.3.1",
     "deepmerge": "^4.2.2",
     "framer-motion": "^1.11.1",
     "postcss": "^7.0.36",
     "postcss-loader": "^3.0.0",
     "postcss-preset-env": "^6.7.0",
     "react-helmet": "^6.1.0",
-    "ts-xor": "^1.0.8"
+    "ts-xor": "^1.0.8",
+    "uuid": "^8.3.2"
```

Comment:
I wonder what happens if you uninstall this from here and components but install it on root. I wonder if it would do the lerna thing and apply it to the package that needs it 🤔